server/mailrepository/mailrepository-api/src/main/java/org/apache/james/mailrepository/api/MailRepositoryUrl.java

```diff
@@ -53,11 +53,11 @@ private MailRepositoryUrl(String value) {
         Preconditions.checkNotNull(value);
         Preconditions.checkArgument(value.contains(PROTOCOL_SEPARATOR), "The expected format is: <protocol> \"" + PROTOCOL_SEPARATOR + "\" <path>");
         this.value = value;
-        String protocol = Splitter.on(':')
+        String protocol = Splitter.on(PROTOCOL_SEPARATOR)
             .splitToList(value)
             .get(PROTOCOL_PART);
         this.protocol = new Protocol(protocol);
-        this.path = MailRepositoryPath.from(value.substring(protocol.length() + PROTOCOL_SEPARATOR.length()));
+        this.path = MailRepositoryPath.from(value.substring(protocol.length()));
```

Comment:
Adding a simple test like:
```

    @Test
    public void getPathShouldReturnValue() {
        assertThat(MailRepositoryUrl.from("proto://abc").getPath())
            .isEqualTo(MailRepositoryPath.from("abc"));
    }
```
shows that this is wrong. And that at least this test is missing.
You can use the second part of the splitted value instead.